### PR TITLE
Js1week1/homework

### PR DESCRIPTION
**Pull request by ReenuMary**

### CODE REVIEW

File: javascript/javascript1/week1/js1homework.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+//Excercise1
+const yearOfBirth = 1980;
+const futureYear = 2025;
+const userName = "Reenu";
+const currentAge = new Date().getFullYear() - yearOfBirth;
+const futureAge = futureYear - yearOfBirth;
+
+console.log(userName + " is " + currentAge + " years old now.");
```
Comment (cecastosic) [inline javascript/javascript1/week1/js1homework.js]: Check Template literals and how you can write these statements in a more elegant way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
I can see you used template literals later in the homework 👍

File: javascript/javascript1/week1/js1homework.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+console.log("She will be " + futureAge + " years old in " + futureYear);
+
+//Excercise2
+//"1 dog year = 7 human years"
+
+const dogYearOfBirth = 2015;
+const dogYearFuture = 2025;
+let shouldShowResultInDogYears = true;
```
Comment (cecastosic) [inline javascript/javascript1/week1/js1homework.js]: Why let?

File: javascript/javascript1/week1/js1homework.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  );
+} else {
+  console.log(
+    `Your dog will be ${dogAgeInFuture} human years old in ${dogYearFuture}`
+  );
+}
+
+//Excercise 3
+//housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+const houseDimensionPeter = [8, 10, 10];
```
Comment (cecastosic) [inline javascript/javascript1/week1/js1homework.js]: Imagine you can put all values in the same array for both Peter and Julia, for example, 
`const volumeInMeters = [8*10*10,5*11*8];` `const gardenSizeInM2 = [100,70];`
where you know that volumeInMeters[0] is Peter's house volume, and another one is Julia's etc.
Can you refactor the rest of your code so you will have fewer lines of code?

File: javascript/javascript1/week1/js1homework.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  "Limitted",
+  "Company",
+  "For You",
+  "Technologies",
+  "Machiens",
+  "Services",
+  "Gateway",
+];
+
+const randomNumber = Math.floor(Math.random() * 10);
```
Comment (cecastosic) [inline javascript/javascript1/week1/js1homework.js]: What if you need two random numbers per array? 🙂

**Review by cecastosic — COMMENTED** [review]
Great job! @ReenuMary 💪  Nice with writing comments and using Prettier. I added some comments that can help you with refactoring. 🌸

File: javascript/javascript1/week1/js1homework.js [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+      ${calculatedHousePricePeter - actualPricePaidPeter}
+       less than the calculated price. `
+  );
+} else if (calculatedHousePricePeter == actualPricePaidPeter) {
+  console.log(
+    `Peter paid ${actualPricePaidPeter}. Same as the calculated price`
+  );
+} else {
+  console.log(
+    `Peter paid
```
Comment (cecastosic) [inline javascript/javascript1/week1/js1homework.js]: There is a strange format of the messages, I am not sure if it was attentional or not.

File: javascript/javascript1/week1/js1homework.js [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+console.log("Calculated price for Peter's house " + calculatedHousePricePeter);
+console.log("Actual price paid for Peter's house " + actualPricePaidPeter);
+
+if (calculatedHousePricePeter > actualPricePaidPeter) {
+  console.log(
+    `Peter paid 
+      ${calculatedHousePricePeter - actualPricePaidPeter}
+       less than the calculated price. `
+  );
+} else if (calculatedHousePricePeter == actualPricePaidPeter) {
```
Comment (cecastosic) [inline javascript/javascript1/week1/js1homework.js]: `==` or `===`?

File: javascript/javascript1/week1/js1homework.js [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+console.log("Calculated price for Julia's house " + calculatedHousePriceJulia);
+console.log("Actual price paid for Julia's house " + actualPricePaidJulia);
+
+if (calculatedHousePriceJulia > actualPricePaidJulia) {
+  console.log(
+    `Julia paid 
+      ${calculatedHousePriceJulia - actualPricePaidJulia}
+       less than the calculated price. `
+  );
+} else if (calculatedHousePriceJulia == actualPricePaidJulia) {
```
Comment (cecastosic) [inline javascript/javascript1/week1/js1homework.js]: `===`

**Comment by ReenuMary** [comment]
Hi Svetlana,
  thank you for the review comments. I've tried to implement all your
comments. Also made an attempt to do a constructor for the house pricing
exercise( kept it in a separate file).
Thank you
Reenu

File: javascript/javascript1/week1/js1homework2.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
+//Excercise 3
+
+function HousePrice(houseDimension, gardenSize, actualPricePaid, userName) {
```
Comment (cecastosic) [inline javascript/javascript1/week1/js1homework2.js:3]: Impressive 👏 I thought about using arrays where the first value would be related to Peter, and the second related to Julia. But this is great also. 🌸

File: javascript/javascript1/week1/js1homework.js [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+  );
+} else {
+  console.log(
+    `Your dog will be ${dogAgeInFuture} human years old in ${dogYearFuture}`
+  );
+}
+
+//Excercise 3
+//housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+const houseVolume = [8 * 10 * 10, 5 * 11 * 8];
```
Comment (cecastosic) [inline javascript/javascript1/week1/js1homework.js:30]: 👏

**Review by cecastosic — APPROVED** [review]

**Pull request merged**